src/Models/Constants/TelemetryConstants.ts

```diff
@@ -0,0 +1,191 @@
+                        countVisualBadgeType: 'countVisualBadgeType',
+                        countVisualColorType: 'countVisualColorType',
+                        parentSceneHash: CUSTOM_PROPERTY_NAMES.ParentSceneHash
+                    }
+                }
+            }
+        }
+    },
+    /** builder section of the app */
+    Builder: {
+        /** scene list page */
+        SceneList: {
+            UserAction: {
+                /** creating a new scene */
+                CreateScene: {
+                    Cancel: {
+                        eventName: 'SceneList.CreateScene.Cancel'
+                    },
+                    Confirm: {
+                        eventName: 'SceneList.CreateScene.Confirm',
+                        properties: {
+                            hasCoordinates: 'hasCoordinates',
+                            hasDescription: 'hasDescription'
+                        }
+                    },
+                    Initiate: {
+                        eventName: 'SceneList.CreateScene.Initiate'
+                    }
+                },
+                /** delete an existing scene */
+                DeleteScene: {
+                    Cancel: {
+                        eventName: 'SceneList.DeleteScene.Cancel'
+                    },
+                    Confirm: {
+                        eventName: 'SceneList.DeleteScene.Confirm'
+                    },
+                    Initiate: {
+                        eventName: 'SceneList.DeleteScene.Initiate',
+                        properties: {
+                            itemIndex: 'itemIndex'
+                        }
+                    }
+                },
+                /** modify an existing scene */
+                EditScene: {
+                    Cancel: {
+                        eventName: 'SceneList.EditScene.Confirm'
+                    },
+                    Confirm: {
+                        eventName: 'SceneList.EditScene.Confirm'
+                    },
+                    Initiate: {
+                        eventName: 'SceneList.EditScene.Initiate',
+                        properties: {
+                            itemIndex: 'itemIndex'
+                        }
+                    }
+                },
+                /** Select a scene from the list */
+                SelectScene: {
+                    eventName: 'SceneList.SelectScene'
+                }
+            }
+        }
+    },
+    /** viewer section of the app */
+    Viewer: {}
+};
+
+/**
+ * The metrics the app can emit
+ * Structure is like this:
+ *  - App region (Builder/Viewer)
+ *    - High level component (SceneList, ElementForm, etc)
+ *      - User action/System metric
+ *        - Event/action
+ *           - eventName: the name of the beacon to emit
+ *           - properties: names of the custom properties that the event will have attached to it
+ */
+export const TelemetryMetrics = { Adapter: {}, Builder: {}, Viewer: {} };
```

Comment:
do you use this const somewhere? I didn't see. did you intend to make it as interface and use for TelemetryEvents above?

Reply:
Not yet, I just made it so it's ready if we want to create metrics in addition to events. They are a specific type of event with different properties. It will be a different collection of beacons that are specifically for that type.